lib/fetch-external.nix

```diff
@@ -0,0 +1,18 @@
+# Provides a function for fetching a GitHub repo from a JSON spec,
+# overridable with the given entry on the NIX_PATH.
+
+{ fetchFromGitHub }:
```

Comment:
Don't import `fetchFromGitHub` if you're not going to use it :)

Reply:
Fixed, thanks.